dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -38,6 +38,19 @@ startsecs=0
 dependent_startup=true
 dependent_startup_wait_for=rsyslogd:running
 
+{% if not skip_chassisd %}
```

Comment:
I feel we should have some flag like we have for fan control HAVE_FANCONTROL_CONF below, to decide whether chassisd should be started ?  This flag inturn could be derived based on properties defined in sonic-buildimage/device/../.. files.

We will be able to achieve the same with skip_chassisd flag, but then with that approach we would need to go and modify/create this entry in all other 1RU skus right ?

Reply:
Agree. Currently, the assumption was we would 'gracefully' exit like pcied.

Chassisd does not have any conf that needs to be read, but should I just create an empty chassisconf file for chassis SKU and set the HAVE_CHASSIS_CONF purely based on presence of the file? I think this is reasonable..

Reply:
I remember there was a file created for chassis based platforms, device/<platform>/chassisdb.conf .. Can we use the presence of this file... than creating a new file ? 
@jleveque for your comments as well

Reply:
@judyjoseph @jleveque The chassisdb.conf is for database connections. Instead of overloading that file, introduced pmon_chassisd.conf which only modular chassis SKUs can include. In future, this file can evolve into providing additional chassis or module behavior. For example, if certain states like offline or fault are supported etc.

Reply:
@mprabhu-nokia: What do you mean, "Instead of overloading that file"? All we need to do is check if it's available. Why create a new pmon_chassisd.conf file? As long as we can guarantee that chassisdb.conf will ONLY exist for modular chassis and it will exist for ALL modular chassis, we can use it, and we do not need to create an extra file. Am I missing something?

Reply:
@jleveque @judyjoseph For now, will go ahead and use chassisdb.conf. My assumption was that all vendors might not need to use the chassisdb.conf, it was tied to DB container, might not be required in Line-card etc. In future, if the feature warrants a new conf file, we can change it.